### PR TITLE
Switch to system fonts

### PR DESCRIPTION
**Pull request by TheTripleV**

### CODE REVIEW

**Comment by TheTripleV** [comment]
Github, Medium, and Bootstrap use system fonts.

**Comment by Daltz333** [comment]
I'm not the hugest fan here of these
- Look is inconsistency across OS's and browsers. Looks particularly bad on Windows Firefox and on my Ubuntu linux.
- There are some issues, Android Firefox does not appear to have mono inline-code.
- Lato and Fontawesome are still being downloaded, so not really a positive here.

File: source/_static/css/frc-rtd.css [hunk 1]
```diff
@@ -123,6 +123,20 @@
...
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI",    Roboto,              Oxygen-Sans, Ubuntu, Tahoma,      Droid Sans,   Helvetica, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+	/*            Apple           Chrome macOS      Win >= Vista   Android + ChromeOS,  KDE          Guess   Win <= XP    old android   backup     default      apple emojis         win emojis        wingdings?*/
+}
+
+.rst-content .toctree-wrapper>p.caption, h1, h2, h3, h4, h5, h6, legend {
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI",    Roboto,              Oxygen-Sans, Ubuntu, Tahoma,      Droid Sans,   Helvetica, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+}
+
+.highlight * {
+	font-family: Menlo, Cascadia Mono, Consolas, Monaco, Liberation Mono, Lucida Console, monospace;
```
Comment (Daltz333) [inline source/_static/css/frc-rtd.css:136]: I prefer `Consolas` over `Menlo` and `Cascadia Mono` here

File: source/_static/css/frc-rtd.css [hunk 2]
```diff
@@ -123,6 +123,20 @@
...
 
+* {
+    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI",    Roboto,              Oxygen-Sans, Ubuntu, Tahoma,      Droid Sans,   Helvetica, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+	/*            Apple           Chrome macOS      Win >= Vista   Android + ChromeOS,  KDE          Guess   Win <= XP    old android   backup     default      apple emojis         win emojis        wingdings?*/
+}
+
+.rst-content .toctree-wrapper>p.caption, h1, h2, h3, h4, h5, h6, legend {
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI",    Roboto,              Oxygen-Sans, Ubuntu, Tahoma,      Droid Sans,   Helvetica, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+}
+
```
Comment (Daltz333) [inline source/_static/css/frc-rtd.css:134]: If we keep "Segoe UI". We should increase weight of the font when inside headers and the sidebar to something thicker, perhaps `600`

image
vs
image

**Comment by Daltz333** [comment]
TODO: Minify Fontawesome

**Comment by Daltz333** [comment]
Blocker on this PR is that RTD still seems to force the download of some fonts. Need to modify theme

**Comment by TheTripleV** [comment]
Also, need to add in Windows 11's font SegoeUI Variable. It has two variations for text and headings, SegoeUI Variable Text and SegoeUI Variable Display, respectively so the heading font definition and body font definitions need to be separated.

**Comment by TheTripleV** [comment]
It doesn't download Lato and Roboto Slab anymore.

**Comment by Daltz333** [comment]
Pending merge after FRC season (late April early May)

**Review by Daltz333 — APPROVED** [review]

**Pull request merged**